Makefile

```diff
@@ -1,6 +1,6 @@
 .DEFAULT_GOAL := build
 
-.PHONY: build generate vet test clean fmt licenses verify-licenses
+.PHONY: build generate tidy vet test clean fmt licenses verify-licenses
```

Comment:
Question: what is the purpose of `generate` target? I saw it only declared as a phony target, but it doesn't seem to have any use. cc @ahmed-mez @knusbaum @jonbodner

Reply:
It's likely an oversight when we first created the Makefile? Perhaps it was copied from another Go project that has a generate target. As far as Orchestrion is concerned, we don't have any code generation yet. You can get rid of it if you want.

Reply:
As it's a minor nit, I'll keep it. Maybe the idea was to run `go:generate` commands? I see one usage of `stringer` in `event.go.`